### PR TITLE
Introductory docs for `dapp-manager`

### PR DESCRIPTION
**Pull request by shadeofblue**
to accompany those for the dapp runner:
* https://github.com/golemfactory/dapp-runner/pull/40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,200 @@
...
+Golem. It works together with the [dapp-runner](https://github.com/golemfactory/dapp-runner/).
+While the responsibility of the latter is to run a single Golem application, `dapp-manager` takes
+care of spawning, interacting with, and stopping the running instances of the `dapp-runner`.
+
+## Quick start
+
+### Yagna daemon
+
+As the `dapp-manager` uses the `dapp-runner`, which in turn requires a properly configured
+[yagna](https://github.com/golemfactory/yagna) daemon, you'll need to have it set up.
```
Comment (nzmel) [inline README.md:13]: "you'll seen to have it set up" feels out of place in this sentence

File: README.md [hunk 2]
```diff
@@ -1,8 +1,200 @@
...
+
+For now, please follow the ["Requestor development: a quick primer"](https://handbook.golem.network/requestor-tutorials/flash-tutorial-of-requestor-development)
+tutorial and ensure that your `yagna` is up and running. Only the first part of this
+tutorial is required - you don't need to run the blender example.
+
+Most importantly, make sure you have set the `YAGNA_APPKEY` in your evironment.
```
Comment (nzmel) [inline README.md:19]: environment is mispelled

**Review by stranger80 — CHANGES_REQUESTED** [review]
Good stuff. Minor suggestions added.

File: README.md (hunk 1 above)
Comment (shadeofblue) [reply README.md:13]: @nzmel what do you mean, it's "need", not "seen" and it's a requirement to have the yagna daemon set-up before you're able to run `dapp-manager` ... what's out of place?

File: README.md (hunk 2 above)
Comment (shadeofblue) [reply README.md:19]: good catch 👍

File: README.md [hunk 3]
```diff
@@ -1,8 +1,200 @@
...
+you have initialized the payment driver with `yagna payment init --sender`,
+and that you have set the `YAGNA_APPKEY` environment variable.
+
+Then run:
+
+```bash
+dapp-manager start --config sample_config.yml dapp-store/apps/webapp.yaml
+```
+
+The app is started in a background `dapp-runner` process, and you're returned an application id in
```
[the remark this answers is not in the document]
Comment (shadeofblue) [reply README.md:76]: okay, making it capitalized

File: README.md [hunk 4]
```diff
@@ -1,8 +1,200 @@
...
+deploy the specified applications on Golem.
+
+### Stop / Kill
+
+The `stop` and `kill` commands terminate the given `dapp-runner` instance, the main difference
+being the signal that's sent to do that. Essentially, `stop` should be enough and should give the
+`dapp-runner` a chance to shut the app down gracefully, correctly terminating the services,
+closing the agreements and paying for them.
+
+In case the `stop` is stuck for whatever reason, you might want to resort to `kill` which terminates
```
[the remark this answers is not in the document]
Comment (shadeofblue) [reply README.md:152]: indeed, I think I wanted to say "the stop command" ;)

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (shadeofblue) [reply README.md:76]: 👍

File: README.md (hunk 2 above)
Comment (shadeofblue) [reply README.md]: updated

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (shadeofblue) [reply README.md]: updated

File: README.md [hunk 5]
```diff
@@ -1,8 +1,200 @@
...
+and that you have set the `YAGNA_APPKEY` environment variable.
+
+Then run:
+
+```bash
+dapp-manager start --config sample_config.yml dapp-store/apps/webapp.yaml
+```
+
+The app is started in a background `dapp-runner` process, and you're returned an application id in
+the form of a hexadecimal string. You can use this id to query the state and other output streams
```
[the remark this answers is not in the document]
Comment (shadeofblue) [reply README.md]: updated

**Comment by shadeofblue** [comment]
@stranger80 @nzmel I have responded to your comments and/or updated the document, please re-review

**Review by stranger80 — APPROVED** [review]

**Pull request merged**